app/tests/optional_test.cc

```diff
@@ -168,10 +164,7 @@ TEST_F(OptionalTest, CopyConstructor) {
   EXPECT_TRUE(another_copy_of_optional_int.has_value());
   EXPECT_EQ(another_copy_of_optional_int.value(), 9999);
 
-  SetupExpectCall()
-    .Construct(1)
-    .CopyAndMove(2, 1)
-    .Destruct(4);
+  SetupExpectCall().Construct(1).CopyAndMove(2, 1).Destruct(4);
```

Comment:
As mentioned above, I think this is a case where having the manual formatting is arguably superior (though I feel less strongly about this one). Another // clang-format off/on?

Reply:
Done.

Also found a few more.